docs/Freestyle Tuning Principles.md

```diff
@@ -0,0 +1,285 @@
+entire flight ([BF doc ref](https://github.com/betaflight/betaflight/wiki/4.2-Tuning-Notes#dynamic-battery-sag-compensation)).
+By enabling VBat Sag Compensation the craft will fly more consistently
+and predictively.
+
+If you plan to use this feature it's crucial to enable it before
+performing the PID tuning.
+
+#### Suggested setting: values for a 5"
+
+|   | **Value** |
+| ------------- | ------------- |
+| **VBat Sag Period (vbat_sag_lpf_period)**  | 200 (20 second)|
+| **VBat compensation**    | 40-60|
+
+     
+                       
+
+### PID
+
+PID are at the core of a quad tuning, with PID tuning we can achieve a
+good quadcopter attitude.
+
+D is the most important PID term to achieve smooth flying, D helps to
+minimize prop was as well as dumpening any quads movement. Freestyle
+quads tend to use higher D gains.
+
+To counterbalance a higher than usual D gain P needs to be increased as
+well.
+
+A simple approach to tune P and D is to set a desired D gain (e.g. 45)
+and slowly increase P as high as possible without producing any bounce
+back on flips and rolls (see [UAV Tech video](https://www.youtube.com/watch?v=qK5APBg76AU)).
+
+The I term is generally good enough on default, however if the quad
+feels sloppy increasing the gain could improve the overall attitude.
+
+#### Suggested settings: values for a 5"
+
+
+|   |  **P** | **I** | **D** |
+| ------------- | ------------- | ------------- | ------------- |
+| **Roll**  | 60-70  | 90-100  | 40-50  |
+| **Pitch**  | 06-70  | 90-100  | 40-50  |
+| **Yaw**  | 30-40  | 90-100  | 0  |
+
+                    
+#### Advanced Considerations
+
+**Equal setpoint tracking latency across axes**
+
+To further improve consistency it's important to properly tune all axes
+(Roll, Pitch and Yaw) this will ensure that the setpoint tracking
+latency is equal for each axe.
+
+**Motor max out**
+
+Another big factor for consistency is that when the craft is commanded
+to make a move, the motors should not max out. If that happens you'll
+get completely different responsiveness, and somewhat unpredictably.
+This is again a key factor in consistency: having enough power and
+authority on all axes.
+
+### Feed Forward
+
+Feed Forward is used to help the quad copter tracking the setpoint
+(stick inputs) closer. In other words it reduces the latency between
+stick movement and quadcopter movement.
+
+Feed Forward anticipates how the quadcopter will move based on how fast and in which direction the sticks are moved.
+
+Feed Forward is great to increase
+responsiveness but makes the craft controll less direct (more variables will affect the rotation rate) hence harder for the pilot to pridict how the quad will behave especially on RC link with low frequency frame rate (e.g. Crossfire).
+
+
+#### Suggested setting: off or low
+
+**Unless the quadcopter suffers from severe lag (setpoint tracking
+latency) requiring too high P gains it is advised to turn off Feed
+Forward for freestyle quads.**
```

Comment:
Definitely don't recommend turning OFF feedforward. This will make the quad feel sloppy and lazy on initial stick response. Reducing feedforward from default values might be advisable depending on the users' preference for stick feel.

Reply:
As stated in the principles section making the quad feel sloppy is not necessarily a bad thing for freestyling. That being said I agree the suggestion should be to: Reduce FF or turn it off. (I'll fix that).